src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ProjectItemsSchema.xaml

```diff
@@ -66,12 +66,15 @@
 
     <ContentType Name="XamlAppDef" DisplayName="Workflow definition" ItemType="XamlAppDef" />
 
+    <ContentType Name="Resource" DisplayName="Resource" ItemType="Resource" />
```

Comment:
I wonder aloud if we want to go back and give these things better names to describe what they are used for; Resource (WPF), Application definition (WPF)?

Reply:
I was wondering that too but at least with the Resource case I seem to remember using it in Winforms at least once. From memory it made it an embedded resource with no namespace prefix (as distinct from Embedded Resource), but maybe it doesn't matter if it says WPF in the name anyway

Reply:
I don't believe this used by WinForms, or if you used it in WinForms, it was via the WinFx targets. See FileClassification target/FileClassifier task which handles these in Microsoft.WinFx.targets.

Reply:
You might be thinking of csc's `-resource` switch, which is how `<EmbeddedResource/>` indirectly ends up in the assembly.

Reply:
I must be remembering wrong, I've just checked and the Resource item type does nothing in a standard WinForms app.